optimum/exporters/openvino/__main__.py

```diff
@@ -46,6 +47,24 @@
 logger = logging.getLogger(__name__)
 
 
+def tokenizer_export(
```

Comment:
I think it should be in convert.py file together with other conversion functions.

Reply:
Moved.